website/content/en/docs/golang/references/client.md

```diff
@@ -1,4 +1,8 @@
-# Using the Controller Runtime Client API with Operator SDK
+---
```

Comment:
This whole doc is really about the controller-runtime client. Have we been keeping this up-to-date? Maybe we should pare this way down and provide links to controller-runtime's docs (or if they don't exist, contribute them upstream)?

This is non-blocking.

Reply:
Yes, but I am not much sure on whether the doc is kept updated. Maybe @camilamacedo86 or @estroz could help in here.

Reply:
IMO, this should be outside the scope of this PR. @joelanford could you file a separate issue for this?